geode-core/src/main/java/org/apache/geode/management/internal/configuration/domain/Configuration.java

```diff
@@ -169,7 +170,8 @@ public void toData(DataOutput out) throws IOException {
     // Before 1.12, this code wrote a non-null HashSet of jarnames to the output stream.
     // As of 1.12, it writes a null HashSet to the stream, so that when we can still read the old
```

Comment:
This comment seems outdated since you are now writing the hash set of jar names.